### ISSUE TITLE
Nomarker displayed when using innerHTML feature

### ISSUE THREAD

**Issue by Awazleon** [coryasilva/Leaflet.ExtraMarkers#65]
Hello,

I'm using a specific font to display "Invader" like character. Using innerHTML to set the right font to use, I can't see the marker. Only the character I want to use is correctly display on the map, as well as the marker shadow but no marker.
Here is the code I use : 
var aliveIcon = L.ExtraMarkers.icon({
  markerColor: 'green',
  innerHTML: '<div class="invafont" style="font-size: 1em; color: white; padding-top: .7em;">C</div>',
  shape: 'circle',
  svg: true
});

Moving svg option to false, the marker is well display. I'm surely wrong somewhere in the innerHTML line but I don't know what to add or modify.
You can say, ok, it works, so what ? but I would like to use svg because in that case, the color choice is wider.

regards

**Comment by coryasilva** [comment]
@Awazleon Hey thanks for using this package!  Sorry the API is so abysmal -- I need to clean it up.  

Anyway I think you need to remove the innerHTML definition.  Instead use a css class.  If I am understanding your code, it should look something like this.

```js
var aliveIcon = L.ExtraMarkers.icon({
markerColor: 'green',
extraClasses: 'alive-icon'
shape: 'circle',
svg: true
});

```

```css
/* css class */
.alive-icon {
  content: 'C';
  ...
}

**Comment by Awazleon** [comment]
@coryasilva Thx for your prompt answer.
I tried your solution but unfortunately, it doesn't work for me. The markers are well displayed but without icon.

Anyway, by digging a bit I found the following solution :
var aliveIcon = L.ExtraMarkers.icon({
  icon: 'fa-number',
  markerColor: 'green',
  extraClasses: 'invader-font',
  number: 'C',
  shape: 'circle',
  svg: true
});

associated with the following CSS
.invader-font {
    font-family: invadersregular;
}
